examples/rtl_433_mqtt_hass.py

```diff
@@ -943,6 +964,8 @@ def bridge_event_to_hass(mqttc, topic_prefix, data):
     published_keys = []
 
     base_topic, device_id = rtl_433_device_info(data, topic_prefix)
```

Comment:
Minor nit, but it feels a bit unclean that you didn't remove the old assignment to `base_topic`. 

I know this whole script needs work, but If you don't want to refactor rtl_433_device_info, then just use:

 `_, device_id = rtl_433_device_info(data, topic_prefix)`.  

The underscore seems to be the prevailing convention for a value you don't care about when unpacking a tuple during assignment.